miscellaneous-commands/check-duration.md

```diff
@@ -1,31 +1,40 @@
 # check-duration
 
-<TODO: describe>
+This command is equivalent to the previous version’s `/status` command. It allows you to see how long you can set a temporary action. If you [vote for MoonlightBot](/support/upvote-moonlightbot.md), purchase a [premium subscription](/support/premium.md), [join the support server](https://discord.com/invite/hNQWVVC) (which besides support you can also read news and changelogs, participate in [volunteer programs](/support/volunteering.md) and in giveaways, meet the person who wrote this and the developer behind all this, among more), or if you started using MoonlightBot less than 7 days ago, the durations for how long you can set a temporary action to last change. Refer to the chart to see how long you can set a temporary action for:
+
+| Status              | Maximum Duration    |
+| ----------------- | ---------------- |
+| Default               |7 Days   |
+| Default [(Beta Branch)](/support/beta.md) |14 Days |
+| Joining [the support server](https://discord.com/invite/hNQWVVC) or for your first 7 days |30 Days |
+| Being a [Premium Subscriber (Basic Tier)](/support/premium.md#tiers) |90 Days  |
+| Being a [Premium Subscriber (Advanced and greater Tiers)](/support/premium.md#tiers) |365 Days  |
+| Having [Voted Recently](/support/upvote-moonlightbot.md) |Double your current maximum |
+
+{% hint style="info" %}
+When your maximum allowed duration changes, we recommend that you use [`/list-temproles`](/role-management-commands/list-temproles.md) to check if all of your temporary and pause roles are [sustained](/start-up/faqs.md#how-does-the-temprole-sustain-mechanic-work).
+{% endhint %}
 
 ## Required permissions
 
-MoonlightBot requires the following permissions to successfully execute this command:
+MoonlightBot requires the following permissions to execute this command successfully:
```

Comment:
please leave the autogenerated stuff untouched - if you disagree with the wording of some templates we can discuss it but applying changes only here will result in the pages becoming inconsistent with each other